ApplicationB.methods.chpl:13: In function 'callFoo1':
ApplicationB.methods.chpl:14: warning: X.foo()
ApplicationB.methods.chpl:15: warning: X.bar()
ApplicationB.methods.chpl:27: Function 'callFoo1' instantiated as: callFoo1(arg: owned MyClass)
ApplicationB.methods.chpl:17: In function 'callFoo2':
ApplicationB.methods.chpl:18: warning: X.foo()
ApplicationB.methods.chpl:19: warning: X.bar()
ApplicationB.methods.chpl:28: Function 'callFoo2' instantiated as: callFoo2(arg: owned MyClass)
ApplicationB.methods.chpl:13: In function 'callFoo1':
ApplicationB.methods.chpl:14: warning: Y.foo()
ApplicationB.methods.chpl:15: warning: Y.bar()
ApplicationB.methods.chpl:36: Function 'callFoo1' instantiated as: callFoo1(arg: owned MyClass)
ApplicationB.methods.chpl:17: In function 'callFoo2':
ApplicationB.methods.chpl:18: warning: Y.foo()
ApplicationB.methods.chpl:19: warning: Y.bar()
ApplicationB.methods.chpl:37: Function 'callFoo2' instantiated as: callFoo2(arg: owned MyClass)
X.foo()
X.bar()
X.foo()
X.bar()
Y.foo()
Y.bar()
Y.foo()
Y.bar()
